mypyc/ir/ops.py

```diff
@@ -1281,6 +1281,10 @@ class BinaryIntOp(RegisterOp):
     LEFT_SHIFT = 203  # type: Final
     RIGHT_SHIFT = 204  # type: Final
 
+    # logical
+    NOR = 300  # type: Final
```

Comment:
NOR is somewhat unusual choice here, since it's not directly supported in hardware or C. It would be more standard to implement boolean not using xor, as `value XOR 1`